src/index.js

```diff
@@ -82,14 +113,99 @@ const addTask = (e) => {
   }
 // console.log(localStorage.tasks)
 };
+
 // Clear all tasks
 
 myForm.addEventListener('submit', addTask);
-clearbtn.addEventListener('click', () => {
-  const tasks = document.querySelector('.itemesList');
-  while (tasks.firstChild) {
-    tasks.removeChild(tasks.firstChild);
+clearbtn.addEventListener('click', clearCompletedTask);
+window.addEventListener('DOMContentLoaded', () => {
+  const tasksArr = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+  if (tasksArr.length > 0) {
+    tasksArr.forEach((item) => {
+      const article = document.createElement('article');
+      article.className = 'article';
+      const attribute = document.createAttribute('data-index');
+      attribute.value = item.index;
+      article.setAttributeNode(attribute);
+      article.innerHTML = `<div class="item"><input type="checkbox" class="check-box">
```

Comment:
- [ ] You did an excellent job working on this milestone! 💯 However, there's a minor change you need to implement in order to approve. At the moment, when the page loads, the to-dos don't load the correct status provided by the local storage. This happens because you're not using the `item.status` here.

```suggestion
      article.innerHTML = `<div class="item"><input type="checkbox" class="check-box" checked=${item.status}>
```

Take in mind that this only updates the checked status and not the styling, so please consider implementing it! 🤓 🛠️

Reply:
@anagudelogu  Thanks for your insightful review, kindly explain to me what it means because I failed to understand.